LivingCity/traffic/b18CUDA_trafficSimulator.cu

```diff
@@ -457,14 +458,17 @@ __global__ void kernel_trafficSimulation(
   uchar *laneMap,
   LC::B18IntersectionData *intersections,
   uchar *trafficLights,
-  float deltaTime)
+  float deltaTime,
+  const parameters simParameters)
```

Comment:
It might be a good idea to pass this structure by reference, just in case it grows in the future.